### PR TITLE
Inheart 148 updates

### PR DESCRIPTION
**Pull request by pmusialik-intive**
Fixed those issues:
- star colour is different from the design,
- no "already added to favourite" state, also not considered in the design,
- text to icons proportions are slightly different on author and price.

For now, user favorites functionality is just frontend mock, will be BE in the future, of course.
Reverted registerComponent to Mobile so that it could work on iOS simulator.
Removed x padding from auctions page.
Fixed top safe area.

EDIT: star color and position to decide 😄

image
image

### CODE REVIEW

**Comment by zgr92** [comment]
> Reverted registerComponent to Mobile so that it could work on iOS simulator.

Please test with Android build (not emulator). The name was changed from Mobile to inHEARTive, because of Android deploy to Google Play. Having "Mobile" there was causing the issue with Android build. Are you sure there isn't another fix that will update the name in iOS build or iOS config files to stick to inHEARTive?

File: apps/mobile/src/main.tsx [hunk 1]
```diff
@@ -1,4 +1,4 @@
 import { AppRegistry } from 'react-native';
 import AppRoot from './app/App';
 
-AppRegistry.registerComponent('inHEARTive', () => AppRoot);
+AppRegistry.registerComponent('Mobile', () => AppRoot);
```
Comment (zgr92) [inline apps/mobile/src/main.tsx:4]: This change potentially breaks Android build that is already set up and tested in Github Actions

**Review by zgr92 — CHANGES_REQUESTED** [review]
Please make sure this is not breaking Android build in Github Actions

File: apps/mobile/src/main.tsx (hunk 1 above)
Comment (pmusialik-intive) [reply apps/mobile/src/main.tsx:4]: But previous change broke ios development :D

Comment (pmusialik-intive) [reply apps/mobile/src/main.tsx:4]: Ok we need to find some solution

**Review by konradhanusintive — APPROVED** [review]

**Review by zgr92 — APPROVED** [review]

**Pull request merged**